README.md

```diff
@@ -72,13 +72,13 @@ Manage > Access (IAM) > Access groups > Access policies.
 <!--
 You need the following permissions to run this module.
 
-- Account Management
-    - **Sample Account Service** service
+- IAM Services
```

Comment:
suggesting this for this section. See comment in PR.

You need the following permissions to run this module:

```md
- IAM services
    - **Sample IBM Cloud** service
        - `Editor` platform access
        - `Manager` platform access
- Account management services
    - **Sample account management** service
        - `Editor` platform access
```